Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM 416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-core-runtime:1.0.0
```

Comment:
It would be beneficial to make the image version a build `ARG`, with a default of `latest`.

This way, we can determine which version of the base image is used to build the runtime image in the pipeline, by setting/updating the build argument, without needing to update the Dockerfile each time.

e.g.
```
ARG CORE_VERSION="latest"
FROM 416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-core-runtime:${CORE_VERSION}
```